src/Extensions/MemberExtension.php

```diff
@@ -52,7 +52,7 @@ public function updateCMSFields(FieldList $fields)
             $qrcodeURI = $this->GoogleAuthenticatorQRCode();
             $fields->addFieldToTab('Root.Main', ToggleCompositeField::create(
                 null,
-                'Second Factor Token Secret',
+                _t(self::class . '.CMSTOGGLEQRCODELABEL', 'Second Factor Token Secret'),
```

Comment:
I think it should be `static::class`, `self::class` breaks (Or the other way around, not sure)